### PR TITLE
'Don't show this message again' is broken for dialogue screen

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: go/base/static/js/src/components/views.js [hunk 1]
```diff
@@ -106,6 +106,16 @@
       this.content = options.content;
       this.optional = options.optional;
       this.animate(options.animate);
+
+      this.rebindActions();
+    },
+
+    rebindActions: function() {
```
Comment (justinvdm) [inline go/base/static/js/src/components/views.js]: Any ideas on a better name?

**Comment by hodgestar** [comment]
:+1:

File: go/base/static/js/src/components/views.js (hunk 1 above)
Comment (hodgestar) [inline go/base/static/js/src/components/views.js]: resetActionHandlers?

Comment (justinvdm) [inline go/base/static/js/src/components/views.js]: Sounds good.

**Comment by hodgestar** [comment]
Re :+1:ing. :)

**Pull request merged**